example/viam_example_app/ios/Runner/Info.plist

```diff
@@ -25,11 +25,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>$(FLUTTER_BUILD_NAME)</string>
+	<string>1.0.0</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>$(FLUTTER_BUILD_NUMBER)</string>
+	<string>1.0.0</string>
```

Comment:
With Flutter 3.16 they changed how they label these strings in the apple projects code - we ought to regenerate our project to address this - but for now this is a fine bandaid.

if people copied this code to start there own apps it could eventually cause problems like this.